letsencrypt-compatibility-test/letsencrypt_compatibility_test/util.py

```diff
@@ -25,6 +25,7 @@
 def create_le_config(parent_dir):
     """Sets up LE dirs in parent_dir and returns the config dict"""
     config = copy.deepcopy(constants.CLI_DEFAULTS)
+    config["strict_permissions"] = False
```

Comment:
This is exactly what we should do instead. Could you please piggy back such change or open bug/PR so it's not lost?